docs/content/latest/yugabyte-cloud/cloud-network/cloud-vpc-intro.md

```diff
@@ -0,0 +1,75 @@
+headerTitle: 
+linkTitle: Overview
+description: Requirements and considerations for setting up a VPC network.
+menu:
+  latest:
+    identifier: cloud-vpc-intro
+    parent: cloud-network
+    weight: 10
+isTocNested: true
+showAsideToc: true
+---
+
+A virtual private cloud (VPC) is a virtual network that you can define within a cloud provider. Once you create a VPC on a cloud provider, you can then connect it with other VPCs on the same provider. This is called peering. A VPC peering connection is a networking connection between two VPCs on the same cloud provider that enables you to route traffic between them privately, without traversing the public internet. VPC networks provide more secure connections between resources because the network is inaccessible from the public internet and other VPC networks. In addition, traffic in a VPC network doesn’t count against data transfer costs, as the traffic is all effectively local.
+
+In the context of Yugabyte Cloud, when a Yugabyte cluster is deployed in a VPC, it can connect to an application running on a peered VPC as though it was located on the same network; all traffic stays within the cloud provider's network. The VPCs can be in different regions.
+
+![Peered VPCs](/images/yb-cloud/cloud-vpc-diagram.png)
+
+## Advantages
+
+Deploying your cluster in a VPC network has the following advantages:
+
+- Lower network latency. Traffic uses only internal addresses, which provides lower latency than connectivity that uses external addresses.
+- Better security. Your services are never exposed to the public Internet.
+- Lower data transfer costs. By staying in the provider's network, you won't have any Internet data transfer traffic. (Same region and cross region overages may still apply. Refer to [Data transfer costs](../../../cloud-admin/cloud-billing-costs/#data-transfer-costs).)
+
+## Pricing
+
+There's no additional charge for using a VPC. In most cases, using a VPC will reduce your data transfer costs.
+
+## Locating your VPC
+
+To avoid cross-region data transfer costs, deploy your VPC and cluster in the same region as the application VPC you are peering with.
+
+## Setting the CIDR and sizing your VPC
+
+A VPC is defined by a block of IP addresses, entered in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing). Because you cannot resize a VPC once it is created, you need to decide on an appropriate size before creating it. Calculate how many applications will be connecting to it, and estimate how that is expected to grow over time. You may want to create a large network to cover all contingencies, but a network that is over-sized can impact network performance. If your traffic experiences spikes, you will need to take that into account.
+
+When entering the range for your VPC in Yugabyte Cloud, the size of the network is determined by the prefix length (the number after the `/`). Yugabyte Cloud supports network sizes from `/24` to `/16` as shown in the following table.
```

Comment:
This is incorrect:
For AWS, we will allow them to select CIDR between /26 and /24. This is because, each VPC spans a single region.  
For GCP, we will support two modes (auto and custom). This is similar to what GCP does: https://cloud.google.com/vpc/docs/vpc#subnet-ranges
So, for auto mode, the user can select CIDR between /18 and /16. This is because, we will automatically split the whole range across all supported regions. 
For custom mode, the restrictions will be the same as AWS, where each region in GCP can be allocated a /26 to /24 IP range. 
Let me know if that makes sense :)